pkg/utils/yaml_func_exec.go

```
@@ -0,0 +1,124 @@
+	if err != nil {
+		return err
+	}
+
+	return runner.Run(context.TODO(), parser)
+}
+
+// getNextShellLevel increments the ATMOS_SHLVL and returns the new value or an error if maximum depth is exceeded .
+func GetNextShellLevel() (int, error) {
+	// Create a new viper instance for this operation
+	v := viper.New()
+	v.BindEnv("atmos_shell_level", "ATMOS_SHLVL")
```

golangci-lint (1.64.8)

[error] Error return value of `v.BindEnv` is not checked

(errcheck)